src/Components/ErrorsHandling/ErrorsHandling.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, {
+  useContext,
+} from 'react';
+import cn from 'classnames';
+import { TodoContext } from '../../TodoContext';
+
+export const ErrorsHandling: React.FC = React.memo(() => {
+  const {
+    error,
+    setError,
+  } = useContext(TodoContext);
+
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={cn(
+        'notification is-danger is-light has-text-weight-normal', {
+          hidden: !error,
+        },
+      )}
+    >
+      <button
+        data-cy="HideErrorButton"
+        type="button"
+        className="delete"
+        onClick={() => {
+          setError('');
+        }}
+      />
+      {error === 'Unable to load todos' && (
+        'Unable to load todos'
+      )}
+      {error === 'Title should not be empty' && (
+        'Title should not be empty'
+      )}
```

Comment:
there is no need to write this multiple check

just write ` {error}` and the title iof  error wil be shown